kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -436,8 +436,7 @@ window.ExerciseView = Backbone.View.extend({
 
 });
 
-
-window.ExercisePracticeView = Backbone.View.extend({
+window.ExerciseWrapperBaseView = BaseView.extend({
```

Comment:
Comments explaining which methods are intended to be overwritten or what functionality needs to be implemented by views extending this would be good.